app/src/main/kotlin/org/cru/godtools/ui/dashboard/tools/ToolFilters.kt

```diff
@@ -222,7 +222,7 @@ private fun FilterMenuItem(
 )
 
 @Composable
-private fun FilterMenuItem(
+fun FilterMenuItem(
```

Comment:
make this internal, this function should have no use outside of the tool drop down filters and doesn't need to be exposed as a public API.

It's a good habit to get into to think about the intended usage of functions to determine if it's something that you are intending to be publicly available for everyone or just something that you are using internally in this module only.

Reply:
I'm also not a total fan of using this Composable in a different layout how you are currently using it.

It's introducing a cross-dependency between different composables, if we need to come back and refactor the tools UI and change functionality/UI there it will have a potential impact on a completely different composable. The fact that you needed to increase the visibility to access this method is an indicator that you are potentially using code outside of it's original intended purpose.

The 2 potential alternative solutions are:
- just duplicate this method or use the `ListItem` composable directly in your `LessonsFilter` composable
- re-organize the code so that there is a common "Filters" package for the dashboard that is used by both the tools and lessons UIs